Add RestartTimer

hassanila opened the pull request.

Commit:
Update Facebots.js

Added RestartTimer

Commit:
Update README.md

hassanila97 commented:
iT'S NOT FAILING... JUST SOME LINT THINGHS

camilleeyries commented:
ok... It work ?

Commit:
Added files via upload

hassanila97 commented:
Yes, now it works.... but only on Windows trought the .bat

camilleeyries commented:
I want the bot work with all platform... Sorry but I can't accept this request...

Commit:
Update Facebots.js

Ok, now I added the comment only on Windows and also added test to check which platform we are in so that it restarts only in Windwos....  Merge it plz.

hassanila97 commented:
Check the last commit

camilleeyries commented:
ok but you need to make the .bat file xD

hassanila97 commented:
Made
 already

camilleeyries commented:
ok i will merge it but I don't know why github tell me there are conflicts in branchs...

hassanila97 commented:
Cuz you edited Facebots.js before merging

Commit:
Update Facebots.js

Commit:
Merge remote-tracking branch 'refs/remotes/camilleeyries/master'

# Conflicts:
#	Facebots.js

hassanila97 commented:
MERGE NOW

camilleeyries commented:
Yeah yeah, I fix the conficts

camilleeyries commented:
woohoo !
